pkg/experiment/block/block.go

```diff
@@ -23,29 +23,11 @@ const (
 	symbolsPrefetchSize = 32 << 10
 )
 
-func estimateReadBufferSize(s int64) int {
-	const minSize = 64 << 10
-	const maxSize = 1 << 20
-	// Parquet has global buffer map, where buffer size is key,
-	// so we want a low cardinality here.
-	e := nextPowerOfTwo(uint32(s / 10))
-	if e < minSize {
-		return minSize
-	}
-	return int(min(e, maxSize))
-}
-
-// This is a verbatim copy of estimateReadBufferSize.
-// It's kept for the sake of clarity and to avoid confusion.
-func estimatePageBufferSize(s int64) int {
-	const minSize = 64 << 10
-	const maxSize = 1 << 20
-	e := nextPowerOfTwo(uint32(s / 10))
-	if e < minSize {
-		return minSize
-	}
-	return int(min(e, maxSize))
-}
+const (
+	// Each 2MB translates to an I/O read op.
+	parquetReadBufferSize      = 2 << 20
+	parquetPageWriteBufferSize = 1 << 20
+)
```

Comment:
It does make sense to increase the `parquetReadBufferSize` size further, e.g., 4MB. However, we need to estimate the actual impact on memory consumption.

I'm less confident about `parquetPageWriteBufferSize`.